### ISSUE TITLE
get Location failed when closing app (No implementation found for method start_location_service on channel)

### ISSUE THREAD

**Issue by bulgarian-beast** [Almoullim/background_location#159]
E/flutter ( 9811): [ERROR:flutter/lib/ui/ui_dart_state.cc(209)] Unhandled Exception: MissingPluginException(No implementation found for method start_location_service on channel com.almoullim.background_location/methods)
E/flutter ( 9811): #0      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:175:7)
E/flutter ( 9811): <asynchronous suspension>
E/flutter ( 9811): #1      BackgroundLocation.startLocationService (package:background_location/background_location.dart:24:12)
E/flutter ( 9811): <asynchronous suspension>

**Comment by bulgarian-beast** [comment]
Hello everyone, I love this project and all the work that has been put into it and I would like to propose a new feature to continue to complete it.
More and more applications, especially delivery applications, need to constantly receive the location of delivery drivers for security reasons, or for tracking deliveries (e.g. a delivery driver kills the application by mistake by leaving after accepting a delivery).
Unfortunately, in the case of a killed application, your plugin is no longer able to broadcast the location.

One way around this is to set up a service that runs in parallel.

I did some research to help and this article does a good job of explaining the problem of an app-killing delivery and how to solve it for android:
```
https://medium.com/@ross_12208/android-location-tracking-foreground-background-force-killed-app-accuracy-aspects-24a151629f22
```

It would be great to be able to continue using your plugin and still receive the location when the application is killed.

**Comment by jaca420** [comment]
so you want to track location even if the app is killed? I was just testing it and even with the app open but phone closed, it doesn't trigger that location. I write in the database when location updates. It works if app is open, but when the phone is locked it doesn't write anything.

**Comment by MoralCode** [comment]
> Hello everyone, I love this project and all the work that has been put into it and I would like to propose a new feature to continue to complete it.

Could you file this as a new issue so it can be tracked separately?
